frontend/stubs/list_methods.py

```diff
@@ -0,0 +1,52 @@
+"""Stub file for methods invoked on lists"""
+from typing import TypeVar, List
+
+T = TypeVar("T")
+
+
+def append(l: List[T], e: T) -> None:
```

Comment:
I believe so.

These stub methods are only considered when we have attribute access. However, normal top-level functions will be used as normal function calls, so I don't think they will ever be confused with each other.

I tried this example and it typed correctly:

```python
def append(l, e):
    l.append(e)
```